src/Microsoft.Framework.Runtime/PatternsCollectionHelper.cs

```diff
@@ -21,19 +26,19 @@ public static IEnumerable<string> GetPatternsCollection(JsonObject rawProject, s
             {
                 if (!rawProject.Keys.Contains(propertyName))
                 {
-                    return CreateCollection(projectDirectory, defaultPatterns.ToArray());
+                    return CreateCollection(projectDirectory, defaultPatterns, literalPath);
                 }
 
                 var valueInString = rawProject.ValueAsString(propertyName);
                 if (valueInString != null)
                 {
-                    return CreateCollection(projectDirectory, valueInString);
+                    return CreateCollection(projectDirectory, new string[] { valueInString }, literalPath);
```

Comment:
No. The type of `valueInString` is not `string`. It's and `JsonString`. Here implicit type conversion is involved.